ttfautohint/.SRCINFO

```diff
@@ -1,19 +1,21 @@
 pkgbase = ttfautohint
-	pkgdesc = Provides automated hinting process for web fonts
-	pkgver = 1.8.4
-	pkgrel = 1
-	url = http://www.freetype.org/ttfautohint
-	arch = i686
-	arch = x86_64
-	license = GPL
-	license = custom
-	depends = freetype2
-	depends = harfbuzz
-	depends = qt5-base
-	source = https://download.savannah.gnu.org/releases/freetype/ttfautohint-1.8.4.tar.gz
-	source = https://download.savannah.gnu.org/releases/freetype/ttfautohint-1.8.4.tar.gz.sig
-	validpgpkeys = 58E0C111E39F5408C5D3EC76C1A60EACE707FDA5
-	sha256sums = 8a876117fa6ebfd2ffe1b3682a9a98c802c0f47189f57d3db4b99774206832e1
-	sha256sums = SKIP
+        pkgdesc = Provides automated hinting process for web fonts
```

Comment:
Please do not convert tabs to spaces here. This file should be auto-generated by `makepkg --printsrcinfo` and then left alone.